### DISCUSSION TITLE
log4j

### DISCUSSION THREAD

**Discussion by Bammann77** [logisim-evolution/logisim-evolution#1352] [Q&A] [answered]
Dear development team
Can you confirm, that logism-evolution is not affected by the log4j Security Vulnerability CVE-2021-44832?
It apperas to be save, but the risks are too hign taht i do not want to rely on my kimited java knowledge.

**Comment by MarcinOrlowski** [comment, marked as answer]
LSe is not using log4j: https://github.com/logisim-evolution/logisim-evolution/blob/master/build.gradle.kts#L32

**Reply by maehne** [reply]
The shadow plugin we use during build to assemble the JAR does seem to use log4j. That's why I created issue #1331 asking upgrading our dependencies.
